packages/workshop-app/app/routes/_app+/_exercises+/$exerciseNumber_.$stepNumber.$type.tsx

```diff
@@ -359,7 +360,9 @@ export default function ExercisePartRoute() {
 			const hydrated = useHydrated()
 			return hydrated && touchedFilesDivRef.current
 				? createPortal(
-						<TouchedFiles>{children}</TouchedFiles>,
+						<TouchedFiles appName={data[data.type]?.name}>
```

Comment:
To fix the lint error (and the bug it's warning about), could you extract this to a variable that's used outside of the `useMemo` and reference that here instead, then stick the variable in the dependency array:

```suggestion
						<TouchedFiles appName={appName}>
```

Reply:
done